### PR TITLE
Shop and shop group have now a color property.

### LINKED ISSUE #24891: Database upgrade script incomplete in PS 1.7.8.0-beta.1
#### Describe the bug

I ran the sql script `install-dev/upgrade/sql/1.7.8.0.sql` and expected the database schema to be up to date. But I find that some columns are missing:

    ALTER TABLE `ps_shop_group` ADD `color` VARCHAR(32) NULL AFTER `name`;
    ALTER TABLE `ps_shop` ADD `color` VARCHAR(32) NULL AFTER `name`;

When clicking in the BO on `Modules - Module Manager` I get an exception about the missing columns.

#### Expected behavior

The upgrade script should convert the database schema 1.7.7.4 into 1.7.8.0.

#### Steps to Reproduce

1. On a installation of PS 1.7.7.4, install the files for 1.7.8.0 and run the migration script.
2. Log into the BO. The dashboard displays correctly.
3. Click on any menu entry (that displays the shop selector).

#### Additional information

* PrestaShop version: Upgraded 1.7.74 to 1.7.8.0-beta.1
* PHP version: 7.3

### PR DESCRIPTION
**Pull request by lmeyer1**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop / 1.7.80 beta-1
| Description?      | Fixes an issue in the sql upgrade script for 1.7.8.0 beta.1. Adds missing columns `color` on table shop and shop_group.
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #24891.
| How to test?      | Run the upgrade script `1.7.8.0.sql` manually.
| Possible impacts? | A step closer to a working upgrade.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24945)

### CODE REVIEW

File: install-dev/upgrade/sql/1.7.8.0.sql [hunk 1]
```diff
@@ -76,6 +76,9 @@ UPDATE `PREFIX_product` SET `product_type` = "combinations" WHERE `cache_default
 UPDATE `PREFIX_product` SET `product_type` = "pack" WHERE `cache_is_pack` = 1;
 UPDATE `PREFIX_product` SET `product_type` = "virtual" WHERE `is_virtual` = 1;
 
+ALTER TABLE `ps_shop_group` ADD `color` VARCHAR(32) NULL AFTER `name`;
```
Comment (Progi1984) [inline install-dev/upgrade/sql/1.7.8.0.sql]: Ping @matthieu-rolland

File: install-dev/upgrade/sql/1.7.8.0.sql [hunk 2]
```diff
@@ -76,6 +76,9 @@ UPDATE `PREFIX_product` SET `product_type` = "combinations" WHERE `cache_default
...
+ALTER TABLE `ps_shop` ADD `color` VARCHAR(32) NULL AFTER `name`;
```
Comment (Progi1984) [inline install-dev/upgrade/sql/1.7.8.0.sql]:
```suggestion
ALTER TABLE `PREFIX_shop_group` ADD `color` VARCHAR(32) NULL AFTER `name`;
ALTER TABLE `PREFIX_shop` ADD `color` VARCHAR(32) NULL AFTER `name`;
```

**Review by Progi1984 — COMMENTED** [review]
Some comments

**Comment by kpodemski** [comment]
cc @matthieu-rolland 

this should be done via doctrine updates, right?

**Comment by lmeyer1** [comment]
@kpodemski If these updates should be done via doctrine, they aren't in the current release `1.7.8.0-beta.1`, because of #24886. I'm not able to run the upgrade script `install-dev/upgrade/upgrade.php`.

**Comment by Progi1984** [comment]
@lmeyer1 Could you target the `1.7.8.x` branch ?

**Comment by lmeyer1** [comment]
@Progi1984 Sorry, it's done now.

**Comment by Progi1984** [comment]
@lmeyer1 Aouch, you need to rebase your PR.

**Comment by lmeyer1** [comment]
@Progi1984 Oh yes, I see. It's not so simple...

**Comment by matks** [comment]
> @lmeyer1 Aouch, you need to rebase your PR.

Hi @Progi1984 I am wondering: is it possible to rebase a PR that is targeting `develop` on `1.7.8.x` instead? Should we rather recommend cherry-pick?

The following scenario is doable:
- author makes a Pull Request that targets 1.7.8.x
- actually it should target develop
- rebase PR branch on develop

But
- author makes a Pull Request that targets develop
- actually it should target 1.7.8.x
- rebase PR branch on 1.7.8.x
⬆️ 
I am not sure this is possible because develop is always ahead of 1.7.8.x . AFAIK git rebase _adds_ commits on your branch, not remove them.

**Pull request closed without merging**